src/db.ts

```diff
@@ -185,8 +185,18 @@ export class Db {
     return this.s.options;
   }
 
-  // slaveOk specified
+  /**
+   * slaveOk specified
```

Comment:
Is this method deprecated because it's a part of the `Db` class and removing it would be considered a breaking change?

Reply:
It's only used in one place by us and that's for internal use

Reply:
I deprecated this one, because it's part of the public Db class. Please let me know if I didn't get that right. :-)

Reply:
I think you're right!  Just unfortunate we're not able to remove it entirely because it's part of a public api

Reply:
I opened NODE-3818 to track removing this option in the next major release